website/docs/about/overview.md

```diff
@@ -7,43 +7,45 @@ order: 101
 - **Patterns**: repetitive combinations of components and foundational styles that yield more complex functions and features
 - **Documentation**: education, issue tracking, contribution, content management, processes
 
-In Helios we structure our components, icons, and design tokens in a [monorepo](https://github.com/hashicorp/design-system) using yarn workspaces and publish a collection of libraries in [Figma](https://www.figma.com/files/team/1030156573400567478) for designers.
+We structure our components, icons, and design tokens in a [monorepo](https://github.com/hashicorp/design-system) using yarn workspaces and publish a collection of libraries in [Figma](https://www.figma.com/files/team/1030156573400567478).
 
-## Why use the Helios Design System?
+## Benefits of using Helios
 
-While increasing visual and functional consistency across products is the most visibly notable result of design system adoption, the intrinsic value a design system brings can result in complimentary benefits:
+While increasing visual and functional consistency across products is the most visibly notable result of design system adoption, other benefits include the following:
 
 - Increased velocity in the design and development of new products and features
-- Dismantling of silos between departments that adopt and use the system
+- Dismantling silos between departments that adopt and use the system
 - More time spent on solving complex problems, less time spent on duplicative work rebuilding small atoms and molecules
-- Components, foundations, and usage guidelines published by the the Helios Design System meet or exceed WCAG 2.1 AA conformance out of the box in accordance with our [accessibility statement](/about/accessibility-statement).
+- Components, foundations, and usage guidelines published by the the Helios Design System meet or exceed WCAG 2.1 AA conformance out of the box in accordance with our [accessibility statement](/about/accessibility-statement)
 
-## Our consumers
+## Who is Helios for
 
-HashiCorp develops and maintains a number of different products with varying sub-brands and approaches to the user experience. In an effort to bring these different product lines and teams towards a common visual language and single source of truth, the Helios Design System was born.
+To bring the different product lines and teams towards a common design language, Helios is available and recommended for HashiCorp product teams. We work closely with these teams on integration and implementation while collecting feedback to understand consumer needs better and inform our [roadmap](https://go.hashi.co/hds-rollout).
 
-Helios has been adopted at varying levels by all products within the R&D organization. We work closely with these teams at the integration and implementation level while collecting feedback to better understand consumer needs and inform our [roadmap](https://go.hashi.co/hds-rollout).
+Helios does not currently support HashiCorp marketing teams. 
```

Comment:
isn't developer a "marketing" team?

Reply:
I chatted with Brayden about this a bit, and they're in a weird spot with the team being split across design/eng. I'll wordsmith this a bit more.